### PR TITLE
Cleancap

### PR DESCRIPTION
**Pull request by 0xRoM**
A plugin to remove caps if don't contain either handshake or PMKID

## Motivation and Context
requested by @Br-at-d on slack

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/evilsocket/pwnagotchi/blob/master/CONTRIBUTING.md) guide
- [n] I have signed-off my commits with `git commit -s`

### CODE REVIEW

**Comment by evilsocket** [comment]
there are handshakes that hashcat can read but aircrack can not, they would be removed ...

**Comment by evilsocket** [comment]
ahahah well played 👍

**Comment by 0xRoM** [comment]
I also did not realise that, modified to better suit :)
lol, thank you

File: pwnagotchi/plugins/default/AircrackOnly.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    if todelete == 0:
+        result = subprocess.run(('/usr/bin/aircrack-ng '+ filename +' | grep "PMKID" | awk \'{print $2}\''),shell=True, stdout=subprocess.PIPE)
+        result = result.stdout.decode('utf-8').translate({ord(c) :None for c in string.whitespace})
+        if result:
+            logging.info("[AircrackOnly] contains PMKID")
+        else:
+            todetele = 1
+
+    if todelete == 1:
+        subprocess.run(('rm '+filename),shell=True,stdout=subpocess.PIPE)
```
Comment (evilsocket) [inline pwnagotchi/plugins/default/AircrackOnly.py]: please use os.remove

Comment (0xRoM) [reply pwnagotchi/plugins/default/AircrackOnly.py]: fixed :)

**Pull request merged**